app/src/main/java/io/homeassistant/companion/android/settings/SettingsPresenterImpl.kt

```diff
@@ -69,6 +70,7 @@ class SettingsPresenterImpl @Inject constructor(
             when (key) {
                 "connection_internal" -> urlUseCase.saveUrl(value ?: "", true)
                 "connection_external" -> urlUseCase.saveUrl(value ?: "", false)
+                "session_timeout" -> integrationUseCase.sessionTimeOut(value.toString().toInt())
```

Comment:
This still needed?

Reply:
Yes, it's still necessary if the user is allowed to choose time in second to session timedout